WooCommerce/Classes/ViewRelated/Reviews/ReviewsViewController.swift

```diff
@@ -251,6 +239,20 @@ private extension ReviewsViewController {
         }
     }
 
+    /// Presents an action sheet on tapping the menu right bar button item.
+    ///
+    @IBAction func presentActionSheet() {
```

Comment:
Nit: Maybe this should be renamed to `presentMoreActions()` so it sounds less generic?

Reply:
You are right. Done in - https://github.com/woocommerce/woocommerce-ios/pull/5833/commits/99c811a3c02b1f1c18c7e7df9cf6ca69da65a82e